src/tests/gfx_gltf.edn

```diff
@@ -0,0 +1,34 @@
+(def timestep (/ 1.0 120.0))
+(def Root (Node))
+
+(defn get-view-transform [time]
+  (->
+   (Float4 0 0.2 0.2 1.0) >= .arm-pos
+   time (Math.AxisAngleY) (Math.Rotation) (Math.MatMul .arm-pos) (ToFloat3) >= .arm-pos-3
+   {:Position .arm-pos-3 :Target (Float3 0 0.025 0)} (Math.LookAt)))
+(defloop test-chain
+  (Setup
+   0.0 >= .time
+
+   (GFX.BuiltinFeature :Id BuiltinFeatureId.Transform) >> .features
+   (GFX.BuiltinFeature :Id BuiltinFeatureId.BaseColor) >> .features
+   (GFX.DrawablePass :Features .features) >> .render-steps
+
+   (GFX.glTF "external/glTF-Sample-Models/2.0/Avocado/glTF-Binary/Avocado.glb") >> .drawables
+
+    ;; Create view
+   {:Position (Float3 0 15 10) :Target (Float3 0 0 0)} (Math.LookAt) >= .view-transform
+   (GFX.View :View .view-transform) >= .view)
+  (GFX.MainWindow
```

Comment:
empty line after `(Setup)` makes it more readable.

Reply:
Moved to a def and added newlines after Setups